common/src/main/java/org/apache/drill/exec/metrics/CpuGaugeSet.java

```diff
@@ -63,7 +72,7 @@ public OperatingSystemLoad(OperatingSystemMXBean osBean) {
 
   @Override
   public Double getValue() {
-    return osMXBean.getSystemLoadAverage();
+    return (osMXBean != null) ? osMXBean.getSystemLoadAverage(): null;
```

Comment:
What you would need braces here?

Reply:
For readability. Helps delineate the condition and the 2 values.

Reply:
I am OK with parentheses, but please add a space before `:`.